packages/cli/src/commands/network/contracts.ts

```diff
@@ -67,18 +67,6 @@ export default class Contracts extends BaseCommand {
       }
     )
 
-    const tokenBalanceColumns: ux.Table.table.Columns<(typeof contractInfo)[number]> = {}
```

Comment:
In accordance with the principle that we dont want to give favored treatment to mento tokens now that they are not core of celo, We no longer display the balances each contract holds for of them